gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
```

Comment:
What approach did you use to upgrade the gradle wrapper? Follow this approach rather than just updating this file directly: https://tomgregory.com/how-to-update-gradle/

Reply:
Hi John, Will that not just update the gradle version on my machine locally, but not necessarily? Others wanting to run this in their own may have different gradle versions